src/OIOProvider/index.js

```diff
@@ -0,0 +1,48 @@
+// =======================================================
+// OIOProvider
+// All OIO components must be nested inside an OIOProvider
+// You can nest OIOProvider inside other OIOProvider
+// This component will be a Context provider in future PR
+// =======================================================
+
+import React, { Component } from 'react'
+import { injectGlobal } from 'emotion'
+import PropTypes from 'prop-types'
+import resetStyles from './resetStyles'
+
+// Reset Styles
+injectGlobal`${resetStyles}` // eslint-disable-line no-unused-expressions
```

Comment:
Could we put this in `componentDidMount` so normalization happens when the `OIOProvider` is actually used, as opposed to when it's simply imported? Or will that cause issues?